upf/upf_pfcp.c

```diff
@@ -1019,6 +1019,7 @@ pfcp_disable_session (upf_session_t * sx)
     upf_pfcp_session_stop_urr_time (&urr->time_threshold, now);
     upf_pfcp_session_stop_urr_time (&urr->time_quota, now);
     upf_pfcp_session_stop_urr_time (&urr->traffic_timer, now);
+    upf_pfcp_session_stop_urr_time (&urr->quota_validity_time, now);
```

Comment:
I would put this after time_quota.

Side Note: it is somewhat strange that the quota_holding_time and monitoring_time is not cleared as well. But that would be a topic for another PR.

Reply:
I believe UPG dont support holding/monitoring timers

Reply:
ahh, right, holding is not supported and monitoring is not a timer (its just a time)